DigitalLearningSolutions.Web.Tests/Helpers/DisplayStringHelperTests.cs

```diff
@@ -182,6 +182,46 @@ public void GetSortableFullName_returns_correctly_formatted_name_with_email_when
             result.Should().Be("FirstName LastName (email@email.com)");
         }
 
+        [Test]
+        public void GetNameWithEmailForDisplay_returns_correctly_formatted_name_for_active_admin()

```

Comment:
This name for this test, and the two new ones after it, don't match the method named used.